learningresources/views.py

```diff
@@ -42,6 +52,11 @@ def create_repo(request):
         form = RepositoryForm(data=request.POST)
         if form.is_valid():
             repo = form.save(request.user)
+            assign_user_to_repo_group(
```

Comment:
Is signals https://docs.djangoproject.com/en/1.8/topics/signals/ a  "better" way to do this ?    /cc @carsongee @ShawnMilo